### ISSUE TITLE
CAID and CURIES for genomic deletions / CNVs

### ISSUE THREAD

**Issue by karafecho** [biolink/biolink-model#1121]
**Is your feature request related to a problem? Please describe.**

The October 2022 QotM revealed confusion related to [CAids](https://onlinelibrary.wiley.com/doi/full/10.1002/humu.23637), in general, and [CAID:CA358802](http://reg.clinicalgenome.org/redmine/projects/registry/genboree_registry/by_caid?caid=CA358802), in particular. It was noted that CAID:CA358802 is a valid Biolink prefix. Additionally, it was noted that CAID functions like a Node Normalizer for variants, providing all other names for that entity, in this case, variations across different versioning of the underlying sequences.

However, when applying CAID to genomic deletions / CNVs, apparently CAIDs work for small deletions, CAIDs, but not for large deletions or CNVs, and Translator does not offer another acceptable approach.

**Tag relevant members for discussion**

@sierra-moxon @mbrush : I did not attend the QotM call during which this issue was raised, so I may not be capturing the complete issue. I believe Colleen X. raised the issue and Chris B. responded. Tag all relevant members to this ticket to start a discussion.

**Comment by sierra-moxon** [comment]
Talking with a couple of people offline, I refined the request to this set of issues:

It's often unclear how IDs should be formatted
- [CLINVAR:1333833](https://www.ncbi.nlm.nih.gov/clinvar/variation/1333833/?oq=1333833&m=NM_152296.5(ATP1A3):c.2552A%3EG%20(p.Gln851Arg)) or ClinVarVariant:1333833?

- [x] remove ClinVarVariant prefix https://github.com/biolink/biolink-model/pull/1129/files

- [DBSNP:rs869320661](https://www.ncbi.nlm.nih.gov/snp/rs869320661) or something different (dbSNP, remove the "rs")?

Prefix registry entries exist in all three flavors for `dbsnp` - good software exists to help normalize prefixes: https://github.com/biopragmatics/bioregistry

In general, we want to follow the prefix guidelines established by Biolink.  Biolink uses `DBSNP` to prefix this identifier, and we can use the tools above to normalize the prefixes as necessary.  If Biolink is missing a prefix (in the id_prefixes section of an entity, please open a ticket and we will get it in. 

- [CAID:CA358802](http://reg.clinicalgenome.org/redmine/projects/registry/genboree_registry/by_caid?caid=CA358802) or remove the "CA" part?

CA is part of the local identifier for this resource, do not remove it. 
also of use: https://github.com/biopragmatics/bioregistry/issues/647

- Is the biolink-model prefix ORPHANET or [ORPHA:2131](https://hpo.jax.org/app/browse/disease/ORPHA:2131)?
- [x] update the prefix in Biolink to 'orphanet' to follow bioregistry designation.  https://github.com/biolink/biolink-model/pull/1128
